src/main.js

```diff
@@ -103,7 +103,16 @@ var load = loader({
   server: {
     requires: ['cfg', 'router', 'docs'],
     setup: ({cfg, router, docs}) => {
-      let hooksApp = app(cfg.server);
+      let hooksApp = app({
+        port:                   cfg.server.port,
+        env:                    cfg.server.env,
+        forceSSL:               cfg.server.forceSSL,
+        trustProxy:             cfg.server.trustProxy,
+        contentSecurityPolicy:  cfg.server.contentSecurityPolicy,
+        robotstxt:              cfg.server.robotstxt,
```

Comment:
This should be `robotsTxt` (capital t).  Or better yet, it can be omitted here since it is not in `config.yml`, and defaults to true.  Same for `contentSecurityPolicy`.

Reply:
Okay .